source/lex.tex

```diff
@@ -296,15 +315,138 @@
 \begin{bnf}
 \nontermdef{universal-character-name}\br
     \terminal{\textbackslash u} hex-quad\br
-    \terminal{\textbackslash U} hex-quad hex-quad
+    \terminal{\textbackslash U} hex-quad hex-quad\br
+    named-universal-character
 \end{bnf}
 
+\pnum
 A \grammarterm{universal-character-name}
+of the form \textbackslash u \grammarterm{hex-quad} or \textbackslash U
+\grammarterm{hex-quad} \grammarterm{hex-quad}
 designates the character in the translation character set
 whose UCS scalar value is the hexadecimal number represented by
 the sequence of \grammarterm{hexadecimal-digit}s
 in the \grammarterm{universal-character-name}.
 The program is ill-formed if that number is not a UCS scalar value.
+
+\pnum
+A \grammarterm{universal-character-name} that is a
+\grammarterm{named-universal-character} designates the
+character named by its \grammarterm{n-char-sequence}. A character is so named if the
+\grammarterm{n-char-sequence} is equal to
+\begin{itemize}
+\item the associated character name or associated character name alias specified in
+ ISO/IEC 10646 subclause ``Code charts and lists of character names'' or
+ \item the control code alias given in \tref{lex.charset.aliases}.
+\end{itemize}
+
+\begin{note}
+The aliases in \tref{lex.charset.aliases} are provided for control characters which
+otherwise have no associated character name or character name alias. These
+names are derived from the Unicode Character Database's \tcode{NameAliases.txt}. For
+historical reasons, control characters are formally unnamed.
+\end{note}
+
+\begin{note}
+None of the associated character names, associated character name aliases, or
+control code aliases have leading or trailing spaces.
+\end{note}
+
+\begin{LongTable}{Control Code Aliases}{lex.charset.aliases}{ll}
+\\ \topline
+\lhdr{Code point} & \rhdr{Control Code Alias} \\ \capsep
+\endfirsthead
+\continuedcaption \\
+\hline
+\lhdr{Code point} & \rhdr{Control Code Alias} \\ \capsep
+\endhead
+\ucode{0000} & \unamec{NULL} \\ \rowsep
+\ucode{0001} & \unamec{START OF HEADING} \\ \rowsep
```

Comment:
I've thought a little more about that. I think the aliases given here are just that: fragments that might appear in source code, thus we should just use `\tcode` directly. Please search/replace and remove the definition of the `\unamec` macro.